src/datatrove/pipeline/filters/gopher_quality_filter.py

```diff
@@ -66,10 +69,9 @@ def filter(self, doc: Document) -> bool | tuple[bool, str]:
         Returns: False if sample.text does not pass any of the the heuristic tests
 
         """
-        from nltk.tokenize import word_tokenize
-
         text = doc.text
-        words = word_tokenize(text)  # TODO we should use language id filter
+        language = doc.metadata.get("language", "en")
```

Comment:
To keep backward compatibility, English tokenizer is used if no language metadata is provided. We could also use [multi-language](https://spacy.io/models/xx) tokenizer from Spacy or require that language metadata is present.